cmake/ctest/drivers/atdm/ride/drivers/Trilinos-atdm-white-ride-cuda-opt-Pascal60.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ "${Trilinos_TRACK}" == "" ] ; then
+  export Trilinos_TRACK=ATDM
+fi
+export ATDM_CONFIG_QUEUE=rhel7G
```

Comment:
@fryeguy52, I was hopping that `ATDM_CONFIG_QUEUE` could get set automatically in the `<system_name>/environment.sh` file from the `ATDM_CONFIG_KOKKOS_ARCH` value parsed out of the `JOB_NAME` var.

Reply:
Actually, looking at the implementation of the file `ride/environment.sh`, I think `ATDM_CONFIG_QUEUE` is getting set correctly, so I think this `export` can be removed, no?